lib/galaxy/model/__init__.py

```diff
@@ -1510,7 +1510,13 @@ def __init__( self, history, action, role ):
         self.role = role
 
 
-class Dataset( object ):
+class StorableObject( object ):
+
+    def __init__( self, id, **kwargs):
+        self.id = id
```

Comment:
I think that's something managed by SQLAlchemy, but I'd let others have a say (@dannon ?).